.circleci/config.yml

```diff
@@ -7,32 +7,13 @@ slack-nightly-build-fail-post-step: &slack-nightly-build-fail-post-step
     - slack/notify:
         channel: $SLACK_DEFAULT_CHANNEL
         event: fail
-        custom: |
-          {
-            "text": "",
-            "blocks": [
-              {
-                "type": "section",
-                "text": {
-                  "type": "mrkdwn",
-                  "text": "🔴 Nightly build failed!"
-                }
-              },
-              {
-                "type": "actions",
-                "elements": [
-                  {
-                    "type": "button",
-                    "text": {
-                      "type": "plain_text",
-                      "text": "View Job"
-                    },
-                    "url": "${CIRCLE_BUILD_URL}"
-                  }
-                ]
-              }
-            ]
-          }
+        template: basic_fail-1
+
+executors:
+  go-builder:
+    docker:
+      - image: mslipper/go-builder:latest
```

Comment:
Would `go-builder` be moved into the monorepo?

Reply:
Yep. The code is already here, in `ops/docker/go-builder` and `ops/docekr/slither` respectively.